django/db/models/fields/related_lookups.py

```diff
@@ -9,34 +11,6 @@
 )
 
 
-class MultiColSource:
```

Comment:
I'm thinking maybe we should pull out `MultiColSource` being removed and `Cols` added as a separate commit (so this PR has 2 commits)

Reply:
I don't think we can since we're renaming `MultiColSource` to `Cols`.

Reply:
Ah yes sorry I missed that also uses TupleIn 👍